experimental/sdk/webpubsub-socketio-extension/package.json

```diff
@@ -21,18 +21,17 @@
     "@azure/web-pubsub-express": "1.0.4",
     "async-mutex": "^0.4.0",
     "base64url": "^3.0.1",
-    "engine.io": "6.4.0",
+    "engine.io": "^6.4.2",
     "socket.io": "^4.6.1",
-    "socket.io-adapter": "^2.5.2",
-    "uuid": "^9.0.0"
+    "socket.io-adapter": "^2.5.2"
   },
   "scripts": {
-    "build": "rimraf ./dist && tsc",
-    "compile": "rimraf ./dist && tsc",
+    "build": "npm run clean && tsc",
+    "check-format": "npx prettier --list-different ./src/ --print-width 100",
+    "format": "npx prettier --write . --print-width 100",
     "test": "npm run test:unit",
```

Comment:
do add command `extract-api` and also list the api document in the PR to show what the API surface looks like